typescript/core/src/common/detectVersion.ts

```diff
@@ -13,9 +12,6 @@ export function detectVersion(prefix: DataView): McapVersion | undefined {
   if (prefix.byteLength < DETECT_VERSION_BYTES_REQUIRED) {
     return undefined;
   }
-  if (MCAP_MAGIC.every((val, i) => val === prefix.getUint8(i))) {
-    return "pre0";
-  }
   if (MCAP0_MAGIC.every((val, i) => val === prefix.getUint8(i))) {
     return "0";
   }
```

Comment:
Same comment as I made on #586, this McapVersion concept and version detection should be removed or replaced with something that doesn't reference versions if we only support one version.